crates/bevy_math/src/primitives/dim3.rs

```diff
@@ -243,9 +392,37 @@ impl Primitive3d for ConicalFrustum {}
 /// A torus (AKA donut) primitive.
 #[derive(Clone, Copy, Debug)]
 pub struct Torus {
-    /// The radius of the overall shape
-    pub radius: f32,
-    /// The radius of the internal ring
-    pub ring_radius: f32,
+    /// The inner radius of the torus, the distance
+    /// from the center to the closest edge of the ring
+    pub inner_radius: f32,
+    /// The outer radius of the torus, the distance
+    /// from the center to the farthest part of the ring
+    pub outer_radius: f32,
```

Comment:
The previous names were `ring_radius` and `radius`, which I found to be very confusing. I believe `ring_radius` was actually the minor radius, i.e. half the thickness of the ring? If that's the case, shouldn't `radius` be the major radius, i.e. the distance from the center of the torus to the middle of the ring?

My new torus representation is slightly different: the inner circle radius (the hole radius) and the outer circle radius. This is what both Godot and (unofficially) Unity seem to use, but it's different from Bevy's earlier representation. Is it fine for the primitive, or are there reasons why the earlier way was used? They can still be computed easily with my representation.